components/Select/Select.tsx

```diff
@@ -18,6 +18,7 @@ export interface SelectOption {
 
 interface ExtendedSelectProps extends SelectProps {
   chooseTheme?: 'light' | 'dark';
+  show?: boolean;
```

Comment:
Please handle this outside the Select component.

Reply:
Now handled in parent component